airflow/providers/openlineage/utils/utils.py

```diff
@@ -647,20 +653,6 @@ def get_filtered_unknown_operator_keys(operator: BaseOperator) -> dict:
     return {attr: value for attr, value in operator.__dict__.items() if attr not in not_required_keys}
 
 
-@deprecated(
-    reason=(
-        "`airflow.providers.openlineage.utils.utils.normalize_sql` "
-        "has been deprecated and will be removed in future"
-    ),
-    category=AirflowProviderDeprecationWarning,
-)
-def normalize_sql(sql: str | Iterable[str]):
```

Comment:
Is this part of the provider public API? if so users might use this for custom code in their deployment.

Reply:
I don't think it is a public API. I think we were being nice with this deprecation warning just in case somebody used it. @mobuchowski ?

Reply:
For now, i removed this removal from the PR - simply removing the deprecation warnings for now.